packages/core/src/kernelTemplate.js

```diff
@@ -62,7 +63,7 @@
       stackFiltering: 'verbose',
     }
 
-    lockdown(lockdownOptions)
+    lockdown(Object.assign({}, DEFAULT_LOCKDOWN_OPTIONS, lockdownOptions ?? {}))
```

Comment:
I think @kumavis might need to speak to this, but:

1. Are we allowed to use `Object.assign` here--does `assign` need to be destructured?
2. Why not `{...DEFAULT_LOCKDOWN_OPTIONS, ...lockdownOptions}`?

Reply:
It seems that the pattern elsewhere in LM was using `Object.assign` over spread, happy to do either, just trying to stick to the existing pattern.

Reply:
I'm curious about that pattern myself.  could a holdover from earlier versions of Node.js or not